src/test/java/com/shippo/model/ShippoTest.java

```diff
@@ -3,16 +3,15 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import org.junit.Before;
-
 import com.shippo.Shippo;
 
-public class ShippoTest {
-	static boolean VERBOSE_TESTING = true;
+import org.junit.Before;
 
+public class ShippoTest {
 	@Before
 	public void setAPIKey() {
 		Shippo.apiKey = "shippo_test_cf1b6d0655e59fc6316880580765066038ef20d8";
```

Comment:
This token existed prior, but do we need to include a real token in the library @assislucas ?

Reply:
Unfortunately we do because the tests are hitting the real endpoint, we should probably record the request/responses so we can remove this...

Reply:
Or we can make this an env var which is set in Circle CI.

Reply:
since this repo is visible to the public, I would assume not, otherwise it could be abused